database/seeders/ApiEndpointsSeeder.php

```diff
@@ -6823,6 +6823,30 @@ private function seedSponsoredProjectsEndpoints(){
                         sprintf(SponsoredProjectScope::Read, $current_realm)
                     ]
                 ],
+                [
+                    'name' => 'add-sponsored-project-logo',
+                    'route' => '/api/v1/sponsored-projects/{id}/logo',
+                    'http_method' => 'POST',
+                    'scopes' => [
+                        sprintf(CompanyScopes::Write, $current_realm)
```

Comment:
this is not the correct one
u should use SponsoredProjectScope::Write

Reply:
done